mediation-add-header/Package.md

```diff
@@ -2,9 +2,8 @@
 
 ## Overview
 
-The "Add Header" policy is available out-of-the-box in Choreo and provides the capability to add arbitrary headers to either 
-the request or the response flowing through an HTTP proxy created in Choreo. Adding a header will append the specified value to the 
-list of values if the header already exists.
+This policy provides the capability to add arbitrary headers to either the request or the response. 
+Adding the same header multiple times will not overwrite the existing header values and it will be appended to the list of values if the header already exists.
```

Comment:
Shall we bold the word - not?

Adding the same header multiple times will **not** overwrite the existing header and its values. When you add an existing header, it gets appended to the list of values. For example: <lets add an example>
 Better if we can mention a example.